lib/stages/rawFileWrite.hpp

```diff
@@ -19,6 +20,7 @@
  * @conf base_dir  String. Directory to write into.
  * @conf file_name String. Base filename to write.
  * @conf file_ext  String. File extension.
+ * @conf _num_frames_per_file Integer. No of frames to write into a single file.
```

Comment:
```suggestion
 * @conf num_frames_per_file Integer. No of frames to write into a single file.
```
The config file doesn't include the prefix.